src/Eloquent/EmbedsRelations.php

```diff
@@ -76,4 +83,27 @@ protected function embedsOne($related, $localKey = null, $foreignKey = null, $re
 
         return new EmbedsOne($query, $this, $instance, $localKey, $foreignKey, $relation);
     }
+
+    /**
+     * Determine if an attribute is an embedded relation.
+     *
+     * @param string $key
+     * @return bool
+     * @throws \ReflectionException
+     */
+    private function hasEmbeddedRelation(string $key): bool
+    {
+        if (! method_exists($this, $method = Str::camel($key))) {
```

Comment:
I've never bothered to do this myself, but assumed CS for PHPLIB prohibited such inline assignments in the interest of readability. Is this common within the Laravel package?

Happy to defer to you.